### DISCUSSION TITLE
Make linear progress bar clickable

### DISCUSSION THREAD

**Discussion by chiwa737** [quasarframework/quasar#14136] [Ideas / Proposals] [answered]
So if mouse click some where in the middle of the progress bar accordingly, it will change its value.  which is implemented in vuetify

**Comment by rstoenescu** [comment, marked as answer]
Hi,

The QLinearProgress and QCircularProgress are components with a representational purpose.
What you are looking for is QSlider/QRange (or QKnob for QCircularProgress). QSlider/QRange also fully customizable (check its props).

https://quasar.dev/vue-components/slider
https://quasar.dev/vue-components/range
https://quasar.dev/vue-components/knob
